docs/schedule/examples/flex.md

```diff
@@ -0,0 +1,324 @@
+<a class="pencil-link" href="https://github.com/MobilityData/gtfs.org/edit/main/docs/schedule/examples/flex.md" title="Edit this page" target="_blank">
+    <svg class="pencil" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M10 20H6V4h7v5h5v3.1l2-2V8l-6-6H6c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h4v-2m10.2-7c.1 0 .3.1.4.2l1.3 1.3c.2.2.2.6 0 .8l-1 1-2.1-2.1 1-1c.1-.1.2-.2.4-.2m0 3.9L14.1 23H12v-2.1l6.1-6.1 2.1 2.1Z"></path></svg>
+  </a>
+
+# Demand responsive services
+
+<hr>
+
+Flex is a GTFS extension project which was adopted officially into the GTFS specification in March 2024, its aims to facilitate discoverability of Demand Responsive Transportation Services.
+
+The following example demonstrates how to model different demand responsive service use cases using Flex. **Please note that the following examples are not necessarily an accurate or complete representation of the agencies’ services.**
+
+## On-demand services within a single zone
+
+Demand-repsonsive services can operate within a specific zone, allowing riders to book pickups at any point A within the zone and drop-offs at any point B within the same zone. An example of this is the [Heartland Express Transit](https://www.co.brown.mn.us/heartland-express-transit?view=category&id=56) service in Minnesota, USA.
+
+<sup>[Download Heartland Express example dataset](../../../assets/on-demand_services_within_a_single_zone.zip)</sup>
+
+### Define trips
+
+Heartland Express service hours are as follow:
+
+- Weekdays:
+  - 8:00 AM - 5:00 PM
+  - 6:15 AM – 5:45 PM (New Ulm zone only)
+- Sunday: 8:00 AM - Noon (New Ulm zone only)
+
+The New Ulm city zone is contained within the Brown County zone. To avoid the ["zone overlap constraint"](/schedule/examples/flex/#zone-overlap-constraint) issue, Heartland Express can be defined with four trips:
```

Comment:
This gives a 404; I think the appropriate way to refer to something in the same page is to use `(#zone-overlap-constraint)`

Reply:
Updated! the previous link was working for me when I displayed the changes locally but change it to the suggested link, it should be ok now!